sr_hand_config/2186/controls/host/pwm/sr_edc_joint_velocity_controllers_PWM.yaml

```diff
@@ -0,0 +1,253 @@
+# Copyright 2023 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# Shadow Hand Defines .. Need Fixing as PID values are not correct
+
+# Shadow Hand Defines .. Need Fixing as PID values are not correct
```

Comment:
What is the meaning of this sentence? Do we want to keep this? If so, shouldn't it be rephrased at least?